service/src/main/java/dk/kvalitetsit/stakit/configuration/SendMessageConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package dk.kvalitetsit.stakit.configuration;
+
+import dk.kvalitetsit.stakit.service.MailQueueService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+
+@Configuration
+@EnableScheduling
+public class SendMessageConfiguration {
+    private static final Logger logger = LoggerFactory.getLogger(SendMessageConfiguration.class);
+
+    @Autowired
+    private MailQueueService mailQueueService;
+
+    @Scheduled(fixedRateString = "${CHECK_MESSAGES_FREQUENCY}")
+    public void sendAnnouncementMessages() {
```

Comment:
Vi bliver nok nødt til at sørge for at to instanser ikke kan komme til at køre jobbet på samme tid. Det kan vi bruge ShedLock til. Se evt. i medcom-vdx-sms for detaljer.